server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationDao.java

```diff
@@ -113,6 +113,10 @@ public void setDefaultGroupId(DbSession dbSession, String uuid, GroupDto default
     getMapper(dbSession).updateDefaultGroupId(uuid, requireNonNull(defaultGroupId, "Default group id cannot be null"), system2.now());
```

Comment:
commit title "SONAR-9156 apply org’s default visibility on first analysis of project" is incorrect

project are not created on first analysis, but created either when report is uploaded or when ws api/projects/create is called.

No mention is made of the second case, neither in commit title nor in JIRA ticket nor in unit tests